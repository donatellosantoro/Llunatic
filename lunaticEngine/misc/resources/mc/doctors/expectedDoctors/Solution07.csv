#doctor
npi,name,hospital,email
_L29,_L169,PPTH,house@ppth.com
_L536,_L542,Sacred Hearth,chase@sh.com

#prescription
rxid,patient,npiref
P1,J. Doe,_L536
P2,W. Smith,_L29
P3,M. White,_L29

