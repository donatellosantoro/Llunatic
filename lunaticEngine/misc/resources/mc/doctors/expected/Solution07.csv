#prescription
rxid,patient,npiref
P3,M. White,5556
P1,J. Doe,_L407
P2,W. Smith,5555

#doctor
npi,name,hospital,email
5556,_SKt0-N1(5556),_SKt0-N2(5556),_SKt0-N3(5556)
_L407,_L411,Sacred Hearth,chase@sh.com
5555,_SKt0-N1(5555),_SKt0-N2(5555),_SKt0-N3(5555)
_L16,G. House,PPTH,house@ppth.com
